tekton/tekton_suite_test.go

```diff
@@ -92,6 +92,7 @@ var _ = BeforeSuite(func() {
 		defer GinkgoRecover()
 		Expect(k8sManager.Start(ctx)).To(Succeed())
 	}()
+	Expect(k8sClient.Create(ctx, newSnapshotEnvironmentBinding)).Should(Succeed())
```

Comment:
Feel free to remove this line, there's no need to create the snapshotEnvironmentBinding here.